phys/module_surface_driver.F

```diff
@@ -395,6 +396,7 @@ SUBROUTINE surface_driver(                                         &
    USE module_sf_tmnupdate
    USE module_sf_lake
    USE module_cpl, ONLY : coupler_on, cpl_rcv
+   use module_ra_gfdleta, only: cal_mon_day
```

Comment:
This requires adding dependency in main/depend.common.

Reply:
@weiwangncar Please check if I've added the dependency correctly. I think this the `cal_mon_day` function can be move to some library module instead of radiation module. But let's save it for later.